cloud_governance/policy/aws/ec2_stop.py

```diff
@@ -87,15 +87,15 @@ def __fetch_stop_instance(self, instance_days: int, delete_instance_days: int, s
                     tag_specifications = [{'ResourceType': 'image', 'Tags': tags}]
                     if sign == ge:
                         tag_specifications.append({'ResourceType': 'snapshot', 'Tags': tags})
-                    try:
```

Comment:
Pls add also comment why we comment out this part, so we will know the reason.
"Comment out termination code to reduce the risk of unintended terminations"

Reply:
done